keyboards/crin/readme.md

```diff
@@ -0,0 +1,13 @@
+# Bachoo Crin
+
+Crin is an F-rowless 65%.
+
+* Keyboard Maintainer: [KnoblesseOblige](https://github.com/KnoblesseOblige)
+* Hardware Supported: STM32F303CBT
+* Hardware Availability: TBD
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader to flash new firmware?

Reply:
I have added some instructions to the README for entering bootloader. Please let me know if they are too verbose.